tests/test_sites.py

```diff
@@ -203,12 +203,21 @@ def test_get_courses_for_site(self, course_count):
 
     @pytest.mark.parametrize('ce_count', [0, 1, 2])
     def test_get_course_enrollments_for_site(self, ce_count):
-
+        self.users = [UserFactory() for i in range(3)]
```

Comment:
u can use `EdlyUserFactory` to automatically create `EdlyUserProfile`.

Reply:
Done